bindings/serialize-v8.cc

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "serialize-v8.h"
+#include "serialize.h"
+
+using namespace v8;
+
+class HeapNode : public Node {
+ private:
+  AllocationProfile::Node *node;
+
+ public:
+  HeapNode(AllocationProfile::Node *node) { this->node = node; }
+
+  virtual std::string name() { return *String::Utf8Value(node->name); }
```

Comment:
All of these virtual functions are overrides. Use the `override` keyword. They can be marked `const` as well, I think.

Reply:
Done.